### PR TITLE
Update FAQ regarding ISO 15118-20

### PR DESCRIPTION
**Pull request by FaHaGit**

### CODE REVIEW

File: docs/source/troubleshooting.rst [hunk 1]
```diff
@@ -64,11 +64,11 @@ file or just use a plain text editor.
...
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ISO15118-20 implementation is currently still under development and not yet included in
-our EVerest releases. The implementation is located on a 
-`test branch of the everest-core repository <https://github.com/EVerest/everest-core/tree/testing/iso15118-20>`_.
-As soon as the implementation has been merged to the main branch, we will include this module in the
-next release cycle.
+The required module for ISO 15118-20 has been included in the image since the chargebyte EVerest 1.0.0 release.
+The implementation is still under development and integrated into the image only for test purposes.
+Please not that it is currently not possible to support ISO 15118-20 together with the other
```
Comment (lategoodbye) [inline docs/source/troubleshooting.rst:69]: I think the consequences for our customers are not clear (Do I need two different firmware versions, two images, two configurations?) Except of this I'm fine.

Comment (FaHaGit) [reply docs/source/troubleshooting.rst:69]: Thank you for the feedback, I will take that into account

Comment (barsnick) [inline docs/source/troubleshooting.rst:69]: Typo: note
```suggestion
Please note that it is currently not possible to support ISO 15118-20 together with the other
```

File: docs/source/troubleshooting.rst [hunk 2]
```diff
@@ -64,11 +64,11 @@ file or just use a plain text editor.
...
+protocols DIN 70121 and ISO15118-2. This is due to architectural reasons and will be fixed in a future
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:70]: Didn't we also include the IsoMux module? So basically, it should work, though completely untested by us.

**Review by barsnick — CHANGES_REQUESTED** [review]

File: docs/source/troubleshooting.rst (hunk 2 above)
Comment (FaHaGit) [reply docs/source/troubleshooting.rst:70]: You are right. But we have currently no configuration for it.

Comment (lategoodbye) [reply docs/source/troubleshooting.rst:70]: In this case i suggest to drop the last two sentences.

**Pull request merged**